pub-critic/src/app/components/Review/Review.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+import classes from "./Review.module.scss";
+import { Review, deleteReview } from "api/ReviewsApi";
+import Image from "next/image";
+import user from "assets/user.svg";
+
+import { starMaker } from "utils/starMaker";
+import Link from "next/link";
+import ReviewForm from "components/ReviewForm";
+import { useState } from "react";
+
+interface ReviewProps {
+  isMine?: boolean;
+  review: Review;
+}
+
+export const ReviewCard: React.FC<ReviewProps> = ({ review, isMine }) => {
+  const [editOpen, setEditOpen] = useState<boolean>(false);
+
+  const handleEdit = () => {
+    setEditOpen((prev) => !prev);
+  };
+
+  const handleDelete = async () => {
+    const confirmation = confirm(
+      "Are you sure you want to delete this review?"
+    );
+    if (!confirmation) return;
+    const response = await deleteReview(review.id);
+    if (response) {
+      alert("Review deleted");
+      window.location.reload();
```

Comment:
Using this in react doesn't make a lot sense 🤔 
You should probably refetch without full page reload unless the page is SSR (even then it can be handled better via layouts)

Reply:
:white_check_mark: